### ISSUE TITLE
[Docs RFC] Add a page showing how to use jobs and user defined functions for pc_cron users

### ISSUE THREAD

**Issue by billy-the-fish** [timescale/docs#3480]
We get regular requests from users to add support for pg_cron. I tell them that they can use our jobs / user defined functions capabilities instead and this has always worked well so far.
Can we add some content so if someone searches for timescale pg_cron we have a page that explains that they can use the capabilities in timescaledb and some examples of how to configure the equivalent of pg_cron job with timescale?

Add a page in https://docs.timescale.com/use-timescale/latest/user-defined-actions/ called Sheduling jobs for pg_cron users with a quick explanation of why jobs serve the same purpose as pg_cron, and add a few examples. 

@ramonguiu, can you suggest the best person to supply the examples please.

**Comment by atovpeko** [comment]
Hi @ramonguiu. Are there any key differences that should be highlighted? I am thinking a "user-defined actions vs pg_cron jobs" angle, with a page either saying it's a full equivalent or highlighting key differences, if any, and providing examples.

**Comment by ramonguiu** [comment]
@atovpeko yeah, I was just thinking we could explain what's possible with user defined actions vs pg_cron so the reader can know if user-defined actions will work for them and then a few examples that show how something they would do in pg_cron  is done with user-defined actions.

**Comment by atovpeko** [comment]
@ramonguiu Ok, who should I approach for examples and possible other input?
